granulate_utils/linux/cgroups/cpu_controller.py

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from abc import abstractmethod
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, Optional, Union
+
+from granulate_utils.linux.cgroups.base_controller import BaseController
+from granulate_utils.linux.cgroups.cgroup import CgroupCore, CgroupCoreV1, CgroupCoreV2
+
+
+@dataclass
+class CpuLimitParams:
+    period: int
+    quota: int
+
+
+class CpuControllerInterface:
+    cpu_stat: str
+
+    def __init__(self, cgroup: CgroupCore):
+        self.cgroup = cgroup
+
+    @abstractmethod
+    def get_cpu_limit_period(self) -> int:
+        pass
+
+    @abstractmethod
+    def get_cpu_limit_quota(self) -> int:
+        pass
+
+    @abstractmethod
+    def get_cpu_limit_params(self) -> CpuLimitParams:
+        pass
+
+    @abstractmethod
+    def set_cpu_limit_quota(self, quota: str) -> None:
+        pass
+
+
+class CpuControllerV1(CpuControllerInterface):
+    cpu_stat = "cpu.stat"
```

Comment:
```suggestion
    CPU_STAT_FILE = "cpu.stat"
```

for others as well. It's confusing that a `CpuController` object will have `self.quota` which is not its quota lol (even though it's a class attr, it's available via `self`, and it's const so make it CONST)

Reply:
I'm adding `_INTERFACE` prefix instead of `_FILE`